pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -79,7 +79,8 @@ func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) error {
 			constants.InferenceServicePodLabelKey: isvc.Name,
 			constants.KServiceComponentLabel:      string(v1beta1.PredictorComponent),
 		}),
-		Annotations: annotations,
+		Annotations:     annotations,
+		ResourceVersion: "0",
```

Comment:
I think this should be set by kubernetes.

Reply:
ye, I forgot this.